http/handler.go

```diff
@@ -39,6 +41,9 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/", handleLogical(core, true, nil))
 	mux.Handle("/v1/", handleLogical(core, false, nil))
 
+	// This is probably wrong in quite a few ways - unsure how to register it properly? Something like Mounting, maybe?
```

Comment:
I would look at keeping this in line with the other telemetry options, i.e. it is in the configuration file and not something that the user mounts via Vault commands. This looks good to me but I'd prefer to get feedback from the devs on it.